test/demo.sh

```diff
@@ -17,57 +17,66 @@ export FAROS_DRY_RUN=1 # Set to 0 or comment out to send the event to Faros
 # =============================================================================
 
 # # help
-# ./faros_event.sh --help
+# ../faros_event.sh --help
 
 # # CI (minimum)
-# ./faros_event.sh CI \
+# ../faros_event.sh CI \
 #     --commit "vcs_source://vcs_org/vcs_repo/commit_sha"
 
 # # CI Event  
-# ./faros_event.sh CI \
+# ../faros_event.sh CI \
```

Comment:
Added spaces?

Reply:
It is in a lower directory so I just added another . (../)